docs/getting-started/enterprise/key-connector.mdx

```diff
@@ -47,6 +50,42 @@ libssl was found".
 4. If you are running the Key Connector from a terminal, restart your terminal to make sure the
    updated `.zshrc` settings are applied
 
+</TabItem>
+<TabItem value="arm" label="ARM">
+
+Given that the Key Connector project is based on NET 5 then we need to use the x86_64 version of
+OpenSSL, thus installing x86_64 packages using Homebrew (a guide with several approaches can be
```

Comment:
💭 This gives me pause here. Can we migrate this very small project to .NET 7 and avoid a lot of these compatibility issues with the framework? I don't question wanting the latest OpenSSL being available, but the architecture plug seems outdated.

Reply:
I don't have much knowledge on the project but I guess it should be possible and it would be awesome; just added the docs because I struggled a lot to get it working.

Reply:
My suggestion: merge this, then arrange with Auth Team to migrate it to .NET 7, and once they've done that migration they can come back and update the docs.

Reply:
I discussed some SDK upgrades across the board with EMs yesterday so I hope that'll happen with more comfortability now. I added the Auth team for review here for the moment.

Reply:
Excellent thank you both!

Reply:
I'm hesitant to upgrade this project to .NET 7 with a LTS version of .NET right around the corner (.NET 8). Definitely appreciative of the docs update and I can put something on the board for the .NET 8 upgrade in the future.